docs/roadmap/README.md

```diff
@@ -0,0 +1,89 @@
+---
+reviewed: 2022-03-22
+styleGuideVersion: 0
+---
+
+# Roadmap
+
+A roadmap for what Stately is working on for XState and Stately tools.
+
+## In Progress
+
+What we’re working on right now.
+
+### Editor
+
+- Bug fixes
+- Discover page
+- Inline implementations
+- Entity snapping
+
+## Upcoming Q2 2022
+
+Features planned for April - June 2022.
+
+### XState
```

Comment:
Should we make this more general? cc. @Andarist 

```suggestion
### Open Source
```

Reply:
> Should we make this more general? cc. @Andarist

I was just thinking of using terms that the audience would be familiar with. But if it’s not all XState, then Open Source makes sense.

Reply:
I can go either way - actually, perhaps keeping it as XState makes sense 👍